src/containers/Tabs.tsx

```diff
@@ -56,4 +67,38 @@ const TabsContainer = ({ tabsData, defaultActiveKey }: TabsProps) => {
     </StyledTabs>
   );
 };
+
+export const LinkTabsContainer = ({
+  tabsData,
+  defaultActiveKey,
+}: LinkTabsProps) => {
+  const navigate = useNavigate();
+  const { i18n } = useTranslation();
+
+  const onTabSelect = (eventKey: unknown) => {
+    navigate(eventKey + "?lang=" + i18n.resolvedLanguage);
+  };
+
+  return (
+    <StyledTabs
```

Comment:
StyledTabs component duplicated, it is possible to use one component. This situation also with LinkTabsDataProps and LinkTabsProps